src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+type TodosProps = {
+  todo: Todos
+};
+
+export const TodoInfo:React.FC<TodosProps> = ({ todo }) => (
+  <>
+    <p className="todo_title">{todo.title}</p>
```

Comment:
use BEM (everywhere)

```suggestion
    <p className="todo__title">{todo.title}</p>
```